app/assets/javascripts/components/RestMap.jsx

```diff
@@ -0,0 +1,44 @@
+class RestMap extends React.Component {
+
+
+
+    componentDidMount() {
+        this.gmaps = new GMaps({
+            el: '#map',
+            lat: 32.075773,
+            lng: 34.783386
```

Comment:
Can be interesting to get start position from [Geolocation API](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation) with fallback to constant values.  :)